index.js

```diff
@@ -1,7 +1,18 @@
+// Create entry point
 const express = require('express');
 const app = express();
 const port = 3000;
-
+// import routes
 const postsRoutes = require('./routes/posts');
 
-app.use("/posts", postsRoutes);
\ No newline at end of file
+// use postsRoutes
+app.use('/posts', postsRoutes);
+
+// get request to homepage
+app.get('/', (req, res) => res.send('main page - will show all posts'));
```

Comment:
Is there a reason why this is also here? I think it's enough to have in the router file.

Reply:
this is the the root endpont = the homepage ;-)